pontoon/teams/views.py

```diff
@@ -168,41 +172,72 @@ def ajax_permissions(request, locale):
 
 @login_required(redirect_field_name='', login_url='/403')
 @require_POST
-def request_projects(request, locale):
-    """Request projects to be added to locale."""
-    slug_list = request.POST.getlist('projects[]')
-    locale = get_object_or_404(Locale, code=locale)
+def request_item(request, locale=None):
+    """Request projects and teams to be added."""
+    user = request.user
+    # Requesting a project
+    if locale:
+        slug_list = request.POST.getlist('projects[]')
+        locale = get_object_or_404(Locale, code=locale)
+
+        # Validate projects
+        project_list = (
+            Project.objects.visible()
+            .filter(slug__in=slug_list, can_be_requested=True)
+        )
+        if not project_list:
+            return HttpResponseBadRequest('Bad Request: Non-existent projects specified')
 
-    # Validate projects
-    project_list = (
-        Project.objects.visible()
-        .filter(slug__in=slug_list, can_be_requested=True)
-    )
-    if not project_list:
-        return HttpResponseBadRequest('Bad Request: Non-existent projects specified')
+        projects = ''.join('- {} ({})\n'.format(p.name, p.slug) for p in project_list)
 
-    projects = ''.join('- {} ({})\n'.format(p.name, p.slug) for p in project_list)
-    user = request.user
+        mail_subject = u'Project request for {locale} ({code})'.format(
+            locale=locale.name, code=locale.code
+        )
+        mail_body = u'''
+        Please add the following projects to {locale} ({code}):
+        {projects}
+        Requested by {user}, {user_role}:
+        {user_url}
+        '''.format(
+            locale=locale.name, code=locale.code, projects=projects,
+            user=user.display_name_and_email,
+            user_role=user.locale_role(locale),
+            user_url=request.build_absolute_uri(user.profile_url)
+        )
+
+    # else requesting a locale
+    else:
+        form = LocaleRequestForm(request.POST)
+        if not form.is_valid():
+            if form.has_error('code', 'unique'):
+                return HttpResponse('This team already exists.', status=409)
+            return HttpResponseBadRequest(form.errors.as_json())
+
+        code = form.cleaned_data['code']
+        name = form.cleaned_data['name']
+
+        mail_subject = u'New team request: {locale} ({code})'.format(
+            locale=name, code=code
+        )
+        mail_body = u'''
+        Please add team {locale} ({code}) to Pontoon.\n
+        Requested by {user}, {user_role}:
+        {user_url}
```

Comment:
You should dedent those 3 lines, otherwise they are indented in the email as well.